File: src/main/java/org/dicio/skill/FallbackSkill.java
package org.dicio.skill;

import org.dicio.skill.chain.InputRecognizer;

public abstract class FallbackSkill extends Skill {

    @Override
    public InputRecognizer.Specificity specificity() {
        return InputRecognizer.Specificity.low; // useless
    }

    @Override
    public float score() {
        return 0; // useless
    }
}

File: src/main/java/org/dicio/skill/FallbackSkill.kt
package org.dicio.skill

import org.dicio.skill.chain.InputRecognizer.Specificity

abstract class FallbackSkill(
    correspondingSkillInfo: SkillInfo,
) : Skill(
    correspondingSkillInfo,
    Specificity.LOW, // useless
) {
    override fun score(): Float {
        return 0.0f // useless
    }
}
